README.md

```diff
@@ -29,9 +29,16 @@ CHT Sync has been specifically designed to work in both local development enviro
 - `Docker`
 - An `.env` file containing the environment variable placeholders from the `.env.template` file. The file should be located in the root directory of the project or set by the operating system. The variables should be customized accordingly for the specific deployment needs.
 
+#### Environment variable
+We have four environment variable groups in the `.env.template` file. It is important to understand the difference between them.
+1. Postgresql and Postgres: Are used to establish the Postgres database to synchronize CouchDB data. It also define the schema and table names to store the CouchDB data. The main objective is to define the environment where the raw CouchDB data is going to be copy to.
```

Comment:
```suggestion
1. Postgresql and Postgres: Are used to establish the Postgres database to synchronize CouchDB data. They also define the schema and table names to store the CouchDB data. The main objective is to define the environment where the raw CouchDB data will be copied.
```

Reply:
@lorerod I don't see a `Postgresql` variable in the `.env.template`. Where do we use that variable? Am I missing something?

Reply:
Both applications use the same set of environment variables.

Reply:
Aaah makes sense, thanks for the explanation!

Reply:
There is this last suggestion to review/solve before I can approve. Can you please look into it, @lorerod ? It's a typo suggestion.

Reply:
Yes! Thank you so much, Andra, for looking into this.